src/main/java/Eln.java

```diff
@@ -0,0 +1,47 @@
+
+import java.util.Scanner;
+
+
+public class Eln {
+    static String LINE = "____________________________________________________________";
+    private static void echoUser() {
```

Comment:
The method name "echoUser" does not match this method's function. Perhaps rename it to something else?
```suggestion
    private static void handleCommands() {
```